android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -417,7 +417,7 @@ private void downloadPageId() {
         }
     }
 
-    private void setupConnection() {
+    public void setupConnection() {
```

Comment:
This works but it's not a good practice. :-/

Reply:
Should I move it to an utils class and make it a static method?

Reply:
@heysadboy I tried doing that but there is a long list of chained calls to different methods all of which are private to MainAcitivity and non-static. Some of them also affect the UI. I am facing problems transferring all this to a Utils class. How should I go about it?
Thanks :)